### PR TITLE
[Analytics] Remove AnalyticsSwift 

### PR DESCRIPTION
**Pull request by ncooke3**

### CODE REVIEW

File: FirebaseAnalytics/README.md [hunk 1]
```diff
@@ -1,2 +1,111 @@
-This directory open sources select files from the Firebase Analytics SDK. Note
```
Comment (paulb777) [inline FirebaseAnalytics/README.md:1]: We probably still want something like these first two lines.

File: FirebaseAnalytics/README.md [hunk 2]
```diff
@@ -1,2 +1,111 @@
...
-that there is no open source infrastructure to build or package them.
+# Firebase Analytics Swift SDK
```
Comment (paulb777) [inline FirebaseAnalytics/README.md:1]: Change title

**Review by paulb777 — APPROVED** [review]

File: FirebaseAnalytics/README.md (hunk 1 above)
Comment (ncooke3) [reply FirebaseAnalytics/README.md:1]: GitHub isn't picking up the git mv very well. These two lines were moved to a README.md in the new Sources sub-directory: https://github.com/firebase/firebase-ios-sdk/pull/13056/files#diff-98fb7621db2b12dd007a7101be7f7de33b98663f976f76bec7c41f067f978b47

File: FirebaseAnalytics/README.md (hunk 2 above)
Comment (ncooke3) [reply FirebaseAnalytics/README.md]: Done in https://github.com/firebase/firebase-ios-sdk/pull/13056/commits/8422ef6e081eae7a69c28d0528e4c4d6ea6cfdee

**Pull request merged**